src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -55,28 +55,54 @@ public boolean extractQuestionDetails(
         List<String> rubricSubQuestions = new ArrayList<String>();
         List<List<String>> rubricDescriptions = new ArrayList<List<String>>();
         
-        int numActualChoices = 0;
-        int numActualSubQuestions = 0;
-        
         // Get list of choices
+        rubricChoices = getRubricChoices(requestParameters, numOfRubricChoices);
+        int numActualChoices = rubricChoices.size();
+        
+        // Get list of sub-questions
+        rubricSubQuestions = getSubQuestions(requestParameters, numOfRubricSubQuestions);
+        int numActualSubQuestions = rubricSubQuestions.size();
+        
+        // Get descriptions
+        rubricDescriptions = getRubricQuestionDescriptions(requestParameters, numOfRubricChoices, numOfRubricSubQuestions);
+        
+        // Set details
+        setRubricQuestionDetails(numActualChoices, rubricChoices,
+                numActualSubQuestions, rubricSubQuestions, rubricDescriptions);
```

Comment:
Indentation looks odd here. See if you can indent `numActualSubQuestions` at the same level as `numActualChoices`. Also consider still putting it on the previous line as line 70 is not that long.